packages/components/pages/f-account-info/src/components/DeleteAccount.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+    <div>
+        <hr>
+
+        <p>{{ $t('deleteAccountMessage') }}</p>
+
+        <f-link
+            is-distinct
+            href="/account/deactivate"
+            target="_blank">
+            {{ $t('deleteAccountLink') }}
+        </f-link>
+    </div>
+</template>
+
+<script>
+import FLink from '@justeat/f-link';
```

Comment:
Linked to my other comment – I'm guessing the styles should be included here instead?

Reply:
No, I registered them in the smart component so that I didn't need to include them in two sub components